package.json

```diff
@@ -47,10 +44,8 @@
     "metrics-graphics": "2.15.6",
     "moment": "2.24.0",
     "mousetrap": "1.6.3",
-    "neutrino": "9.0.0-rc.3",
     "ng-text-truncate-2": "1.0.1",
     "numeral": "2.0.6",
-    "popper.js": "1.15.0",
```

Comment:
I think popper is used by react-strap. Have you tested dropdowns and tooltips to make sure they still work?

Reply:
Yes, it is needed, however, it is not needed as a peer dependency.

Before this change we were installing `popper.js` twice. Once as a peer-dependency and another one as part of `reactstrap`'s dependencies.

This dropdown seems to work:
<img width="126" alt="image" src="https://user-images.githubusercontent.com/44410/63171245-246ef180-c009-11e9-98bd-d72fb7f0e06b.png">

This tooltip seems to work:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/44410/63171273-351f6780-c009-11e9-9bb0-163e8f067bcf.png">

This also seems to work:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/44410/63171289-44061a00-c009-11e9-90b3-3671d2e59e17.png">

```shell
Armens-MacBook-Pro:treeherder armenzg$ npm ls popper.js
treeherder@ /Users/armenzg/repos/treeherder
└─┬ reactstrap@7.1.0
  └─┬ react-popper@0.10.4
    └── popper.js@1.15.0
```

Reply:
I see - makes sense.